src/Components/Register/ManagerCustomerRegistration.js

```diff
@@ -0,0 +1,302 @@
+                while(creditString[0] === "," || creditString[0] === " ") {
+                    creditString = creditString.substring(1);
+                }
+                var cardNumber = creditString.substring(0,10);
+
+                //if the credit card has any characters besides just numbers
+                if(cardNumber.match(/^[0-9]+$/) == null) {
+                    window.alert("Invalid credit card format");
+                    return;
+                }
+                cards[count] = cardNumber;
+                count++;
+
+                //if the user tries to add more than 5 credit cards
+                if(count >= 5) {
+                    window.alert("You can only add up to 5 credit cards");
+                    return;
+                }
+
+                //new credit card string omitting the previous credit card
+                creditString = creditString.substring(10);
+            }
+            this.setState({creditCardsList: cards});
+            //post method
+        }
+    }
+    
+    render() {
+        const { selectedCompany,
+                selectedState
+             } = this.state;
+        return(
+            <div className="main">
+                <div className="card registrationCard">
+                    <h2 className="card-header">Manager-Customer Registration</h2>
+                    <div className="card-body">
+                        <div className="nameRegister row">
+                            <div className="col-6">
+                                <div className="col-4">
+                                    <label className="registerLabel">
+                                        First Name
+                                    </label>
+                                </div>
+                                <input type="text" name="firstName" onChange={this.handleChange} className="form-control" id="firstName"/>
+                            </div>
+                            <div className="col-6">
+                                <div className="col-4">
+                                    <label className="registerLabel">
+                                        Last Name
+                                    </label>
+                                </div>
+                                <input type="text" name="lastName" onChange={this.handleChange} className="form-control" id="lastName"/>
+                            </div>
+                        </div>
+                        <div className="row">
+                            <div className="col-6">
+                                <div className="col-2">
+                                    <label className="registerLabel">
+                                        Username
+                                    </label>
+                                </div>
+                                <input type="Username" name="Username" onChange={this.handleChange} className="form-control" id="Username"/>
```

Comment:
Replace this with just rendering the UserRegistration component here?

Reply:
I was thinking that, but I just decided to do this instead of passing props and checking the type of user

Reply:
Aight that works.